.github/workflows/ghcr_build.yml

```diff
@@ -0,0 +1,57 @@
+name: Build GHCR images
+
+on:
+  push:
+    branches: 
+      - "master"
+  pull_request:
+    branches: [ "master" ]
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+permissions:
+  contents: read
+  packages: write
+
+jobs:
+  build:
+    name: Build images
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - platform: "linux/amd64"
+            runs_on: "ubuntu-latest"
+            cache_id: amd64
+          - platform: "linux/arm/v7"
+            runs_on: "buildjet-2vcpu-ubuntu-2204-arm"
+            cache_id: arm
+          - platform: "linux/arm64"
+            runs_on: "buildjet-2vcpu-ubuntu-2204-arm"
+            cache_id: arm64
+    runs-on: ${{ matrix.runs_on }}
+    steps:
```

Comment:
I would add a timeout to prevent stuck jobs

Reply:
Done